services/elasticache/generator.yaml

```diff
@@ -53,6 +53,19 @@ resources:
         from:
           operation: DescribeEvents
           path: Events
+      ServiceUpdates:
+        is_read_only: true
+        from:
+          operation: DescribeServiceUpdates
+          path: ServiceUpdates
+      UpdateActions:
+        is_read_only: true
+        from:
+          operation: DescribeUpdateActions
+          path: UpdateActions
+      ServiceUpdateActions:
```

Comment:
Can you tell me what the different between UpdateActions and ServiceUpdateActions is?